modules/helm-disabling-helm-chart-repository-in-the-cluster.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="disabling-helm-chart-repository-in-the-cluster_{context}"]
+= Disabling Helm Chart repository in the cluster
+
+You can disable Helm Charts in the catalog by adding `disabled` property in the `HelmChartRepository` custom resource. 
+
+.Procedure
+
+* To disable a recently added Helm Chart repository by using CLI, add the `disabled: true` flag to the custom resource. For example, to remove an Azure sample chart repository, run:
```

Comment:
Is this different if the Helm Chart repo's been around awhile? If so, you need to add that info. If not:
```suggestion
* To disable a Helm Chart repository by using CLI, add the `disabled: true` flag to the custom resource. For example, to remove an Azure sample chart repository, run:
```